### PR TITLE
Soc 2372

### PR DESCRIPTION
**Pull request by qaga**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java [hunk 1]
```diff
@@ -21,16 +21,17 @@
...
   /**
    * ANONS ON MOBILE SECTION
    */
 
-  @Test(groups = "discussions-anonUserOnMobileCanNotWriteNewPost", enabled = false)
-  @RelatedIssue(issueID = "SOC-2372")
+  @Test(groups = "discussions-anonUserOnMobileCanNotWriteNewPost")
   @Execute(asUser = User.ANONYMOUS)
-  @InBrowser(emulator = Emulator.GOOGLE_NEXUS_5)
+  @RelatedIssue(issueID = "SOC-2730 will fix it")
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java]: it's issue id not comment. Remove the comment part

File: src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java [hunk 2]
```diff
@@ -42,7 +43,7 @@ public void anonUserOnMobileCanNotWriteNewPost() {
   @Test(groups = "discussions-anonUserOnDesktopCanNotWriteNewPost")
   @Execute(asUser = User.ANONYMOUS)
   @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
-  @RelatedIssue(issueID = "SOC-2724, Test manually")
+  @RelatedIssue(issueID = "SOC-2730 will fix it")
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java]: it's issue id not comment. Remove the comment part

File: src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java [hunk 3]
```diff
@@ -42,7 +43,7 @@ public void anonUserOnMobileCanNotWriteNewPost() {
   @Test(groups = "discussions-anonUserOnDesktopCanNotWriteNewPost")
   @Execute(asUser = User.ANONYMOUS)
   @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
-  @RelatedIssue(issueID = "SOC-2724, Test manually")
+  @RelatedIssue(issueID = "SOC-2730, <- will fix it")
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java]: change: SOC-2730, <- will fix it

Comment (qaga) [inline src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java]: done

**Comment by justnpT** [comment]
lgtm

**Pull request merged**